install.sh

```diff
@@ -87,7 +87,7 @@ __install() {
   rm netcat-0.7.1.tar.gz
   cd netcat-0.7.1
   mkdir -p ~/go/src/github.com/ocmdev/rita/nc
-  ./configure --prefix=~/go/src/github.com/ocmdev/rita/nc
+  ./configure --prefix=$HOME/go/src/github.com/ocmdev/rita/nc
```

Comment:
Why would you want to put this under the go source folder?  It seems like it should be under something like `/usr/local/bin`.